libs/services/designer-client-services/src/lib/standard/manifests/xml.ts

```diff
@@ -0,0 +1,210 @@
+import { SettingScope } from '@microsoft/utils-logic-apps';
+import type { OperationManifest } from '@microsoft/utils-logic-apps';
+
+const connector: any = {
+  id: 'connectionProviders/xmlOperations',
+  name: 'xmlOperations',
+  properties: {
+    description: 'XML Operations',
+    displayName: 'XML Operations',
+  },
+};
+
+const settings: any = {
+  secureData: {},
+  trackedProperties: {
+    scopes: [SettingScope.Action],
+  },
+};
+
+export const xmlTransformManifest = {
+  properties: {
+    iconUri: 'https://logicappsv2resources.blob.core.windows.net/icons/xml.svg',
+    brandColor: '#804998',
+    summary: 'Transform XML',
+    description: 'Transform XML using XSLT map.',
+
+    inputs: {
+      type: 'object',
+      required: ['content', 'mapName', 'xsltParameters'],
+      properties: {
+        containerId: {
+          name: 'containerId',
+          title: 'Container Id',
+          summary: 'Function container',
+          description: 'The function container',
+          type: 'string',
+          visibility: 'internal',
```

Comment:
There is no visibility internal anymore in new designer, we will have to use hideInUI attribute to just hide in UI but serialie anyways.